lib/routes/qztc/sjxy/index.ts

```diff
@@ -0,0 +1,121 @@
+import { Data, Route } from '@/types';
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+import timezone from '@/utils/timezone';
+
+const rootUrl = 'https://www.qztc.edu.cn/sjxy/';
+const host = 'www.qztc.edu.cn';
+const name = '泉州师范学院-数学与计算机科学学院 软件学院';
+
+export const route: Route = {
+    path: '/sjxy/:type',
+    categories: ['university'],
+    example: '/qztc/sjxy/1939',
+    parameters: { type: '分类，见下表' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name,
```

Comment:
https://github.com/DIYgod/RSSHub/blob/f5677731e3dbf94c41aeb0ac4f8bb3f1d9d80df9/lib/routes/qztc/namespace.ts#L4 https://github.com/DIYgod/RSSHub/blob/f5677731e3dbf94c41aeb0ac4f8bb3f1d9d80df9/lib/routes/qztc/sjxy/index.ts#L10
No need to repeat namespace name in route name.